src/MongoDB.Driver/SslSettings.cs

```diff
@@ -34,7 +34,7 @@ public class SslSettings : IEquatable<SslSettings>
         private static readonly IEqualityComparer<X509CertificateCollection> __certificateCollectionEqualityComparer = new X509CertificateCollectionEqualityComparer();
 
         // private fields
-        private bool _checkCertificateRevocation = true;
+        private bool _checkCertificateRevocation = false;
```

Comment:
There's another default value you need to change in a different file:

https://github.com/vincentkam/mongo-csharp-driver/blob/CSHARP-2279/src/MongoDB.Driver.Core/Core/Configuration/SslStreamSettings.cs#L53

Reply:
Fixed!